objectPerformance/README.md

```diff
@@ -1,18 +1,30 @@
 # Object Performance Tools
 
 ## Setup of Python environment
-  In the final version of this code no
-  setup will be required. Should the provided
-  Python installation not work for some reason
-  it can be installed locally from `environment.yml`.
-  The documentation on how to do this can be found
-  [here](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#sharing-an-environment).
-  Most importantly you need to have `conda` or `miniconda`
-  installed and then run
+  The code has been tested on lxplus,
+  but it should work on any machine that can access `/eos/`.
 
-  ```
-  conda env create -f environment.yml
-  ```
+### Install miniconda
+  To install `miniconda` run the following commands:
+
+    wget https://repo.anaconda.com/miniconda/Miniconda3-py39_4.12.0-Linux-x86_64.sh
+    sh Miniconda3-py39_4.12.0-Linux-x86_64.sh "${PATH}/miniconda3"
+
+  Make sure you are installing the latest version by
+  checking out the available installers [here](https://docs.conda.io/en/latest/miniconda.html#linux-installers).
+
+### Install the environment for the framework
+  The easiest way to set up the `conda` environment is to
+  use the `environment.yml` file provided.
+  Specify the path to your `miniconda3` installation in `prefix`
+  and run:
+
+    conda env create -f environment.yml
+
+  This will create a new environment named `py310`.
+  Before running the next steps, just activate the environment:
```

Comment:
The environment does not need to be activated to run the next steps. The important thing is that the shebang is pointing to the right Python installation.